test/lib/usd/translators/testUsdExportRoots.py

```diff
@@ -33,14 +33,15 @@ class testUsdExportRoot(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         inputPath = fixturesUtils.setUpClass(__file__)
-
-        filePath = os.path.join(inputPath, "UsdExportRootsTest", "UsdExportRootsTest.ma")
-        cmds.file(filePath, force=True, open=True)
+        cls.filePath = os.path.join(inputPath, "UsdExportRootsTest", "UsdExportRootsTest.ma")
 
```

Comment:
Now that there is a test not using this file, the scene needs to be open by each test. Probably makes the tests more independent and avoid accidental interactions.